lib/prefix.h

```diff
@@ -486,7 +486,7 @@ extern int str_to_esi(const char *str, esi_t *esi);
 extern char *esi_to_str(const esi_t *esi, char *buf, int size);
 extern char *evpn_es_df_alg2str(uint8_t df_alg, char *buf, int buf_len);
 extern void prefix_evpn_hexdump(const struct prefix_evpn *p);
-extern bool ipv4_unicast_valid(const struct in_addr *addr);
+extern bool ipv4_unicast_nexthop_valid(const struct in_addr *addr);
```

Comment:
I don't think I'm following this rename: it increases the footprint of the change, but doesn't actually change anything?

Reply:
Yeah, no functional changes, it's a separate commit.

Reply:
but, you know - why? changing the name of the api forces you to make a bunch of other changes, but the real functional change was to the logic in the bgp code. why wasn't that bgp change ... enough?

Reply:
Okay, I will drop that commit.

Reply:
Removed that rename commit.